Sources/CardPayments/Models/Card.swift

```diff
@@ -29,6 +29,11 @@ public struct Card: Encodable {
 
     /// Optional. The billing address
     public var billingAddress: Address?
+    
+    /// The expiration year and month, in ISO-8601 `YYYY-MM` date format.
+    public var expiry: String {
+        "\(expirationYear)-\(expirationMonth)"
+    }
```

Comment:
We don't want to add `expiry` as a public param, since this doesn't maintain parity with Android. Instead, we want the SDK to for the formatting work and send `expiry` to the orders API internally, using the `expirationDate` and `expirationYear` set by the merchant.

Reply:
See Android https://github.com/paypal/Android-SDK/blob/c18d6d675065496e43661fb80eed421f835d6485/CardPayments/src/main/java/com/paypal/android/cardpayments/CardRequestFactory.kt#L13

Reply:
I see. Thank you.